api/src/main/resources/workbench.yaml

```diff
@@ -1911,9 +1932,16 @@ definitions:
         $ref: '#/definitions/Profile'
       password:
         type: string
+      invitationKey:
+              type: string
+
+  InvitationVerRequest:
+    type: object
+    required:
+      - invitationKey
+    properties:
       invitationKey:
         type: string
-        description: Invitation key. Necessary when creating account before public launch.
```

Comment:
Why remove this description?

Reply:
It happened by mistake have added it again